packages/lib-classifier/test/setup.js

```diff
@@ -1,28 +1,45 @@
 import chai from 'chai'
 import chaiDom from 'chai-dom'
 import sinonChai from 'sinon-chai'
-import dirtyChai from 'dirty-chai'
 import { JSDOM } from 'jsdom'
 import fetch from 'node-fetch'
 import nock from 'nock'
+import { afterAll, beforeAll, beforeEach, describe, it, vi } from 'vitest'
+import { expect } from 'chai'
+import { setProjectAnnotations } from '@storybook/react'
+import preview from '../.storybook/preview'
 
 chai.use(chaiDom)
-chai.use(dirtyChai)
 chai.use(sinonChai)
 
+// Vitest uses chai assertion syntax
+global.after = afterAll
+global.before = beforeAll
+global.beforeEach = beforeEach
+global.describe = describe
+global.it = it
+
+/* expect is an exception to the rest of FEM's Vitest config. This is "expect" from the actual
+   chai library because lib-classifier extensively uses sinonChai and chaiDom, and the only way to
+   use sinonChai and chaiDom is to attach them to chai itself, rather than Vitest's implementation.
+*/
+global.expect = expect
+
+// Attachs Story decorator with Grommet theme
+setProjectAnnotations(preview)
+
 // require all net requests to be mocked.
 nock.disableNetConnect()
 nock.enableNetConnect('panoptes-uploads.zooniverse.org')
 
 const jsdom = new JSDOM('<!doctype html><html><body></body></html>', {
-  pretendToBeVisual: true, // See: https://github.com/jsdom/jsdom#pretending-to-be-a-visual-browser
-  url: 'https://example.org/'
+  url: 'https://localhost'
```

Comment:
`pretendToBeVisual` isn't used anywhere else in FEM, so I've changed the `jsdom` config here to match our other FEM packages.